site/docs/v1/tech/apis/_tenant-generator-length-config-rules.adoc

```diff
@@ -0,0 +1,7 @@
+The length of the secure generator used for generating the {generator_display_name}.
++
+If the `{generator_type}.type` == `randomAlpha` then the length must be >= 4 and +<=+ 12. +
+If the `{generator_type}.type` == `randomAlphaNumeric` then the length must be >= 4 and +<=+ 12. +
+If the `{generator_type}.type` == `randomBytes` then the length must be >= 16 and +<=+ 128. +
+If the `{generator_type}.type` == `randomDigits` then the length must be >= 4 and +<=+ 12. +
```

Comment:
This was a move to a common include but I'll make the changes you suggest.

Reply:
So I went with: [field]#{generator_type}.type# (no backticks) as that seemed to be in keeping with the other field references in the tenants page: systemConfiguration.minimumPasswordAge.enabled and 
tenant.passwordValidationRules.breachDetection.onLogin

I also saw that in the User's api for twoFactorDelivery 

I think that leaving it partially qualified is the clearest, because it refers to a sibling property.

I also reworked the verbiage to be a bit clearer.